uniffi_bindgen/src/bindings/swift/templates/RustBufferHelper.swift

```diff
@@ -140,8 +140,13 @@ extension String: Liftable, Lowerable {
         }
         return String(cString: v)
     }
-    func toFFIValue() -> Self {
-        self
+    func toFFIValue() -> UnsafeMutablePointer<CChar> {
+        var rustErr = NativeRustError(code: 0, message: nil)
+        let rustStr = {{ ci.ffi_string_alloc_from().name() }}(self, &rustErr)
+        if rustErr.code != 0 {
+            fatalError("caught a panic while passing a string across the ffi")
```

Comment:
I manually tested this by inserting a deliberate `panic!` on the rust side, then confirming that the swift (and kotlin) side tests correctly surfaced the panic.